docs/architecture/adr-016-protocol-versions.md

```diff
@@ -4,26 +4,24 @@
 
 - How to / should we version the authenticated encryption handshake itself (ie.
   upfront protocol negotiation for the P2PVersion)
```

Comment:
@erikgrinaker does this concern you in any way with the p2p refactor

Reply:
Not really, but it's something we should revisit when we consider switching to [Noise](http://www.noiseprotocol.org). It would depend on what that protocol allows us to do.